tests/console/yast2_lan_device_settings.pm

```diff
@@ -46,12 +48,18 @@ sub run {
     send_key "alt-t";    # select static IP address option
     send_key "tab";
     type_string $static_ip;
+    send_key 'alt-o';
+    wait_screen_change(sub {
```

Comment:
Is it not reliable enough without `wait_screen_change`? `type_string` doesn't do `wait_screen_change` either.

Reply:
Locally, I am not able to see a problem. It was just light precaution since there is a sequence of send_keys without any screen asserting. Which may be a problem in general.

I haven't see any problems with `wait_screen_change` mentioned by @dzedro.

Reply:
We could use send_key_until_needle match and use needle with empty field, that would save us from lost key presses issue too.

Reply:
> We could use send_key_until_needle match and use needle with empty field, that would save us from lost key presses issue too.

Yup, good point. Thanks!

* https://openqa.opensuse.org/tests/1219204#step/yast2_lan_device_settings/22